Lib/ufo2ft/makeotfParts.py

```diff
@@ -180,30 +180,4 @@ def setupFile_featureTables(self):
                     self.outline[tag] = table
 
         elif self.features.strip():
-            if self.font.path is not None:
-                self.features = forceAbsoluteIncludesInFeatures(self.features, self.font.path)
-            fd, fea_path = tempfile.mkstemp()
-            with open(fea_path, "w") as feafile:
-                feafile.write(self.features)
-            addOpenTypeFeatures(fea_path, self.outline)
-            os.close(fd)
-            os.remove(fea_path)
-
-includeRE = re.compile(
-    "(include\s*\(\s*)"
-    "([^\)]+)"
-    "(\s*\))" # this won't actually capture a trailing space.
-    )
-
-
-def forceAbsoluteIncludesInFeatures(text, directory):
-    for match in reversed(list(includeRE.finditer(text))):
-       start, includePath, close = match.groups()
-       # absolute path
-       if os.path.isabs(includePath):
-           continue
-       # relative path
-       srcPath = os.path.normpath(os.path.join(directory, includePath.strip()))
-       includeText = start + srcPath + close
-       text = text[:match.start()] + includeText + text[match.end():]
-    return text
+            addOpenTypeFeatures(self.outline, self.font.path, self.features)
```

Comment:
@adrientetar shouldn't the second argument of `addOpenTypeFeatures` be the path to the feature file itself, rather than the path to the UFO font as you do here?
I understand that a "Font.ufo/features.fea" file may not always be present, but I guess it'd be clearer if in FeatureLibError messages the referenced line/position referred to that, instead of the UFO as a whole.